### PR TITLE
add FreeType 2.6 to ports

### PR DESCRIPTION
**Pull request by anlambert**

### CODE REVIEW

File: embuilder.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
         sdl2-image
         vorbis
         zlib
+        freetype
```
Comment (kripken) [inline embuilder.py]: please keep this list sorted

File: tools/ports/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import bullet, libpng, ogg, sdl, sdl_image, vorbis, zlib
+import bullet, libpng, ogg, sdl, sdl_image, vorbis, zlib, freetype
```
Comment (kripken) [inline tools/ports/__init__.py]: please keep this list sorted

**Comment by kripken** [comment]
Looks good, just 2 small comments.

**Comment by anlambert** [comment]
Woops did not notice that the ports list was sorted, fixed

**Comment by kripken** [comment]
Thanks, please just squash the commits.

**Comment by anlambert** [comment]
Commits squashed

**Comment by kripken** [comment]
Thanks!

**Pull request merged**